scripts/apps/ingest/directives/IngestSourcesContent.ts

```diff
@@ -594,8 +595,16 @@ export function IngestSourcesContent(ingestSources, api, $location,
                             fulltext: '',
                         };
 
+                        var filter_value = '';
+
+                        if (contentTypes.includes('event')) {
+                            filter_value = 'EVENT';
+                        }
+                        if (contentTypes.includes('planning')) {
+                            filter_value = 'PLANNING';
```

Comment:
If the `contentTypes` includes both `event` & `planning`, this would have to be changed to `filter_value = 'COMBINED'`